docs/types.rst

```diff
@@ -169,6 +169,11 @@ Fixed Point Numbers
 Rational and Integer Literals
 -----------------------------
 
+Integer literals are formed from a sequence of numbers in the range 0-9.
+They are interpreted as decimals. Examples include ``69`` and ``01000000``.
```

Comment:
I think ``01000000`` is a bad example because leading zeroes are usually associated with octal numbers.  We should very probably mention that we do not support octal numbers and leading zeroes still mean base 10.